### DISCUSSION TITLE
Create New-BurntToastNotification by Group or Tag

### DISCUSSION THREAD

**Discussion by sravulapalli** [Windos/BurntToast#216] [Q&A]
Hi,

I'm trying to create Burnt Toast Notifications and  specify  `tagId` or `groupId` and use those IDs to remove notifications using `Remove-BTNotification`  command under a specific group or tag. Currently, I see the `New-BurntToastNotification` only has a `UniqueIdentifier` field which can be used to remove notification, but unfortunately you can't group notifications based on UniqueIdentifier (since specifying same identifier replaces a notification) and I see there are no other params like group or tag that you can use to group new notifications under. I read the documentation and I couldn't find anything. Here's a sample of what I'm trying to achieve - 

// Create first Notification
args := []string{
		"-Command",
		fmt.Sprintf(`$title = "%s"; $message = "%s"; New-BurntToastNotification -UniqueIdentifier "unique123" -Group "group123" -Text $message`, title, message),
	}
	cmd = exec.Command("powershell.exe", args...)

// Create another notification under the same group
args := []string{
		"-Command",
		fmt.Sprintf(`$title = "%s"; $message = "%s"; New-BurntToastNotification -UniqueIdentifier "unique456" -Group "group123" -Text $message`, title, message),
	}
	cmd = exec.Command("powershell.exe", args...)

// Delete all notifications under a group (this should delete  both notifications created under group123)
	args := []string{
		"-Command",
		fmt.Sprintf(`$GroupId = "%s";Remove-BTNotification -Group $GroupId`, GroupId),
	}

	cmd = exec.Command("powershell.exe", args...)

**Comment by Windos** [comment]
Hello, and thank you for your contribution to BurntToast.

Today marks the official release of [BurntToast v1.0.0](https://github.com/Windos/BurntToast/releases/tag/v1.0.0). As part of this major milestone and to ensure the project starts its next chapter with a clean slate, I am declaring "issue bankruptcy."

This decision is explained in more detail in the official announcement post: [Ten Years Toasting: BurntToast Hits the Big v1!](https://toastit.dev/2025/07/22/burnttoast-v1/)

As a result, I am closing all outstanding issues and pull requests (and discussions), including this one. Please do not interpret this as a lack of interest in the problem, suggestion, or topic you've raised. This is a necessary step to clear a very old backlog and focus development efforts on the current state of the module.

**If this topic is still relevant to you:**

1.  Please **open a new discussion**. You are welcome to link back to this one for historical context.

I hope this reset will allow me to be more responsive to community feedback going forward.

Thank you for your understanding and for being part of the BurntToast journey.
